src/modules/common/__fixtures__/mockCommandParametersSchemas.js

```diff
@@ -2389,54 +2389,6 @@ export const mockCommandParametersSchemas = {
           },
         },
       },
-      {
```

Comment:
why is this removed?

Reply:
isnt that renamed to `transferCrossChain` ?